tests/samples/yaml/epics.ibek.yaml

```diff
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=file://schemas/ibek.schema.json
+module: epics
+definitions:
+  - name: EPICS_CA_MAX_ARRAY_BYTES
```

Comment:
We have the ability to add arbitrary dbpf's but it seems not arbitrary env_vars. Would this be a good place to put them?

Reply:
If so this should be in addtion to EPICS_CA_MAX_ARRAY_BYTES and any other variables that we expect the majority of IOCs to set.

Reply:
Sure, I think this is a good place